conf/application.conf

```diff
@@ -4,4 +4,20 @@ play.application.loader=AppLoader
 
 play.crypto.secret="Replace me, please!"
 
-panda.system="atom-workshop"
\ No newline at end of file
+panda.domain="local.dev-gutools.co.uk"
+
+panda.authCallback="https://atomworkshop.local.dev-gutools.co.uk/oauthCallback"
```

Comment:
this and panda.domain shouldn't be here any more - should be moved to ~/.configuration-magic